Dockerfile

```
@@ -1,5 +1,5 @@
-ARG PHP_VERSION=7.2
-ARG COMPOSER_VERSION=1.8
+ARG PHP_VERSION=7.3
```

```suggestion
ARG PHP_VERSION=7.4
```
7.4 is currently active version